src/EditorFeatures/Core/Implementation/LanguageServer/AbstractInProcLanguageClient.cs

```diff
@@ -198,11 +198,12 @@ public Task OnServerInitializedAsync()
             ICapabilitiesProvider capabilitiesProvider,
             ILspLogger logger)
         {
-            return new VisualStudioInProcLanguageServer(
+            return new LanguageServerTarget(
                 _requestDispatcherFactory,
                 jsonRpc,
                 capabilitiesProvider,
                 _lspWorkspaceRegistrationService,
+                lspMiscellaneousFilesWorkspace: null,
```

Comment:
matches previous behavior, have https://github.com/dotnet/roslyn/issues/57243 filed for me to followup on this